pkgs/tools/networking/dns-over-https/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "dns-over-https";
+  version = "2.2.2";
+  goPackagePath = "github.com/m13253/dns-over-https";
+  subPackages = [ "doh-client" "doh-server" ];
+  src = fetchFromGitHub {
+    owner = "m13253";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "12xklf7k07vlqbd7xv9p4c6385h874k8g5sagpcg891lib174bad";
+  };
+  vendorSha256 = "07jnrhrn45diw7mgscsiwagsdqbh0p69dnwx5bi10v2jrc66zkjd";
+  goDeps = ./deps.nix;
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/m13253/dns-over-https";
+    description = "High performance DNS over HTTPS client & server";
+    platforms = platforms.unix;
+    maintainers = with maintainers; [ dwoffinden ];
+    license = licenses.mit;
+  };
```

```suggestion
  pname = "dns-over-https";
  version = "2.2.2";

  src = fetchFromGitHub {
    owner = "m13253";
    repo = pname;
    rev = "v${version}";
    sha256 = "12xklf7k07vlqbd7xv9p4c6385h874k8g5sagpcg891lib174bad";
  };
  
  goPackagePath = "github.com/m13253/dns-over-https";

  vendorSha256 = "07jnrhrn45diw7mgscsiwagsdqbh0p69dnwx5bi10v2jrc66zkjd";

  subPackages = [ "doh-client" "doh-server" ];

  meta = with stdenv.lib; {
    homepage = "https://github.com/m13253/dns-over-https";
    description = "High performance DNS over HTTPS client & server";
    platforms = platforms.unix;
    maintainers = with maintainers; [ dwoffinden ];
    license = licenses.mit;
  };
```

Go module does not use deps.nix. Please remove that.